driver-core/src/main/com/mongodb/internal/connection/ClusterableServer.java

```diff
@@ -34,6 +35,15 @@
      */
     void invalidate(Throwable reason);
 
+    /**
+     * Invalidate the description of this server due to the passed in reason.
+     * @param reason the reason for invalidation.
+     * @param sessionContext the session context.
+     *
+     * @since 3.12
```

Comment:
This interface is internal, so no need for @ since annotation

Reply:
I have removed this from the interface.